### DISCUSSION TITLE
Dapper

### DISCUSSION THREAD

**Discussion by hgmauri** [alirezanet/Gridify#93] [Q&A] [answered]
Is it possible to use Gridify with Dapper?

I am currently using Dapper in the Oracle database according to the code below.

```
    public static int? ExecuteAction(string query)
    {
        using var connection = ConnectionManager.GetConnection();
        var result = connection.Execute(query);

        ConnectionManager.CloseConnection(connection);

        return result;
    }

**Comment by alirezanet** [comment, marked as answer]
Hi @hgmauri ,
As far as I know dapper doesn't support LINQ, so the short answer is no. but if you have any other libraries that add LINQ support to dapper you definitely can use Gridify.
